build.zig

```diff
@@ -5,6 +5,7 @@ const Query = std.Target.Query;
 const Mode = std.builtin.Mode;
 
 const config = @import("./src/config.zig");
+const stdx = @import("./src/stdx/stdx.zig");
```

Comment:
Hm, yeah, I'd maybe avoid this! This is not too bad, but, still, it'll cause the build system to read all those stdx file, and then to stat them on each build execution (I _think_ zig parses even unused `@import`s? Not sure here).

Reply:
Or maybe not? It's not like  `@import("std")` would behave differently (which probably means that I am wrong, and Zig won't actually parse more than necessary here). I guess I am just confused, would be happy with either approach!